doc/deprecation_process.md

```diff
@@ -2,57 +2,169 @@ This page can be linked using: [aka.ms/azsdk/python/deprecation-process](https:/
+  - If a migration guide will not be provided.
 
-Do a PR targeting the `main` branch. Post your PR in our [review channel for Python](https://teams.microsoft.com/l/channel/19%3a4175567f1e154a80ab5b88cbd22ea92f%40thread.skype/Language%2520-%2520Python%2520-%2520Reviews?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
+    > # Microsoft Azure SDK for Python
+    >
+    > This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
 
-You're responsible to fix any CI issue related to this PR, if any. Note that `Inactive` packages are disabled from most CI verificiation, therefore the CI should be faster and
-have less requirements.
+  - If the service has been retired and no replacement package exists.
 
-An example of PR can be found the [deprecation of application insights](https://github.com/Azure/azure-sdk-for-python/pull/23024/files)
+    > # Microsoft Azure SDK for Python
+    >
+    > This package is no longer being maintained, as the service has been retired. There is no replacement package for this package.
 
-Once the PR is merged, move to the next step.
+  - If the Azure service no longer exists, a new non-Azure service has replaced it, and existing customers should be directed to the new service's Rest API docs/repo.
 
-# Step 2: Trigger a release 
+    > # Microsoft Azure SDK for Python
+    >
+    > This package is no longer being maintained. Please refer to the samples in the [My New Service repo](https://github.com/microsoft/my-new-service/tree/main) instead.
+    >
+    > For additional support, please open a new issue in the [Issues](https://github.com/microsoft/my-new-service/issues) section of the My New Service repo.
 
-A release here is the same as usual, triggering the release pipeline of you SDK. More instruction can be found at: https://aka.ms/azsdk/release-checklist
+## CHANGELOG.md and _version.py
 
-**Important Note**: As CI don't build Inactive projects right now, you can't build and release. This is a classical chicken and egg problem: we don't want to lose time on testing Inactive projects, but you need at least have one run of build to release the Inactive project. It's important then to pass the variable `"BUILD_INACTIVE=true"` while triggering the release pipeline. But doing this may uncover issues that were not seen in your initial PR as the CI was disabled. To avoid most of those problems, it's highly recommended that you put the commit with "Inactive" LAST in your PR. In other words, push all changes to Readme and ChangeLog, wait to confirm the CI is green, and when everything is clean, push finally "Inactive" in the `setup.py`
+- Update the version in the `azure/mypackage/_version.py` file to the next patch version. This file may be called `version.py` if your package is very old. For example:
+  - If the last released version was 1.2.3b1, the new version should be 1.2.4.
```

Comment:
Is this the recommendation for packages that never reached a stable version? For example, what should a package do that has only been in beta? 1.0.0b1 -> 1.0.0b2?

Reply:
oh good question, will update guidance to mention that they should bump to the next beta in that case.